sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorReceiver.java

```diff
@@ -503,4 +548,93 @@ public String toString() {
         return String.format("connectionId: [%s] entity path: [%s] linkName: [%s]", receiver.getName(), entityPath,
             getLinkName());
     }
+
+    // Temporary type to support both v1 or v2 stack side by side.
+    public static final class ReceiveLinkHandlerWrapper {
```

Comment:
I noticed that there are multiple static classes called ReceivelinkHandlerWrapper. This one is public. Is it possible to move it into a single class file?  Maybe we can consolidate all these implementations.

Reply:
It makes sense to move this temporary type from ReactorReceiver nested to package, as it’s instantiated outside of ReactorReceiver type, also can keep diff in ReactorReceiver minimal.

The other temporary type is in RequestResponseChannel and used only internally to RequestResponseChannel so inner private class. 

These two temporary types have slightly different logic, so I would keep it separate. (will rename the one inner to RequestResponseChannel to `RequestChannelWrapper`)

Reply:
updated as mentioned above.